packages/mui-material/src/TableSortLabel/TableSortLabel.d.ts

```diff
@@ -4,8 +4,46 @@ import { Theme } from '..';
 import { ExtendButtonBase, ExtendButtonBaseTypeMap } from '../ButtonBase';
 import { OverrideProps } from '../OverridableComponent';
 import { TableSortLabelClasses } from './tableSortLabelClasses';
+import { CreateSlotsAndSlotProps, SlotProps } from '../utils/types';
 
-export interface TableSortLabelOwnProps {
+export interface TableSortLabelSlots {
```

Comment:
Please export these overrides and use them in the `SlotProp<>`

```suggestion
export interface TableSortLabelRootSlotPropsOverrides {}

export interface TableSortLabelIconSlotPropsOverrides {}

export interface TableSortLabelSlots {
```